docs/contributor-guide/developer-guide/deploy-system/prerequisits.md

```diff
@@ -3,12 +3,256 @@
+    - `SENDER_EMAIL_ADDRESS="<your_email_address>"`
+    - `STAGING_PROGRAM_ADMIN_NOTIFICATION_MAILING_LIST="<your_email_address>"`
+    - `STAGING_TI_NOTIFICATION_MAILING_LIST="<your_email_address>"`
+    - `STAGING_APPLICANT_NOTIFICATION_MAILING_LIST="<your_email_address>"`
+    - `BASE_URL="https://<your_dev_account_name>.civiform.dev"` (Note: This can be anything you want it to be, but should be similar to something a civic entity might use so it can pass some validation checks in the CiviForm app. Using your dev account name works.)
+    - `STAGING_HOSTNAME="<your_dev_account_name>.civiform.dev"`
+    - `APP_PREFIX="<your_dev_account_name>"` (Note: App prefix allows you to have more than one deployment in the same account. It can be set to whatever you want.)
+    - `SSL_CERTIFICATE_ARN="<arn_of_the_SSL_cert>"` (Find the ARN by clicking into the Certificate ID on the [Certificates page](https://us-east-1.console.aws.amazon.com/acm/home?region=us-east-1#/certificates/list) in AWS.)
+### [cloud-deploy-infra](https://github.com/civiform/cloud-deploy-infra)
+1. Clone [cloud-deploy-infra](https://github.com/civiform/cloud-deploy-infra) onto your local machine.
+2. Set your local copy of `civiform-deploy` to track your local copy of `cloud-deploy-infra`.
+    - In your local copy of `civiform-deploy`, update [this line](https://github.com/civiform/civiform-deploy/blob/4624131e74ff3a417d070f0a34da55744f26c347/bin/lib/checkout.sh#L86) in the `checkout::initialize` function in `checkout.sh` to point to your local repository instead of the remote origin.
+    - Eg. change 
+    ```
+    git remote add origin http://github.com/civiform/cloud-deploy-infra
+    ```
+    to 
+    ```
+    git remote add origin <path_to_local_repo>/cloud-deploy-infra
+    ```
+
+## Setup AWS Nuke
+AWS Nuke is a package that makes it easy to remove all resources from an AWS account. You will want to run this at the end of each working session (at least at the end of each day so we don't leave resources running overnight) and anytime you need to cleanup your AWS account to test new changes. It is destructive, so it's very important to set up your config file correctly to only clear out resources from your dev account. Check out the [README for the `aws-nuke` repository](https://github.com/rebuy-de/aws-nuke) for more information about the inner workings of `aws-nuke`.
+1. Download the correct binary for your machine's architecture/OS from the [aws-nuke repo](https://github.com/rebuy-de/aws-nuke/releases). Or on mac run `brew install aws-nuke`
+2. If you downloaded the binary, unpack the tarball, rename the binary to `aws-nuke` and put it somewhere in your shell's path.
+3. Run `aws-nuke -h` to make sure it is installed correctly.
+4. Create a `nuke.yaml` file and copy the [`nuke.yaml` file used for end to end](https://github.com/civiform/cloud-deploy-infra/blob/main/e2e-test/nuke.yaml) tests this file.
```

Comment:
It would be nice here to have a recommendation for where to put the file for consistency across the team. It will make it easier to track down the file if every someone needs help for example or if someone wants to remove everything they used for civiform development from their machine

Reply:
update: added a recommendation to store it in the root of your local copy of `civiform-deploy`

Reply:
Sounds good. If  we have it inside one of the repositories, you will also need an entry .gitignore.
In that case I'd specify the exact name of the file here (and highlight that it should not be changed),
and add the entry with the name specified here to .gitignore (in a follow up change)

This would be needed for changes that include files in the civiform-deploy repository (e.g. a change that spans both or all three repositories.

Reply:
update: added to `.gitignore` file: https://github.com/civiform/civiform-deploy/pull/35

and updated docs: https://github.com/civiform/docs/pull/267